Makefile

```diff
@@ -74,3 +74,8 @@ makemessages: assets
 
 compilemessages:
 	python -m kolibri manage compilemessages -- -l en > /dev/null
+
+syncmessages:
```

Comment:
I initially strayed away from defining makefile deps that should be run in sequence, since I thought they aren't guaranteed to run in order. However, looks like the Makefile format does guarantee that they do: http://stackoverflow.com/questions/9159960/order-of-processing-components-in-makefile

So I can refactor this to be more granular tasks, with `syncmessages` depending on all of them.